### ISSUE TITLE
[Wikipedia] LaTeX not rendering properly

### ISSUE THREAD

**Issue by shimishimiaeshimiah** [raycast/extensions#15942]
### Extension

https://www.raycast.com/vimtor/wikipedia

### Raycast Version

1.88.4

### macOS Version

15.1.1

### Description

When viewing a Wikipedia article containing LaTeX, the code is not being rendered properly.
![Screenshot 2024-12-18 at 23 52 24](https://github.com/user-attachments/assets/43f6145e-b052-4c82-83c0-a52674855a51)

### Steps To Reproduce

1. Open Raycast
2. Type 'Wikipedia'
3. Press 'Tab'
4. Enter 'Euler-Lagrange equation'
5. Press 'Enter'
6. Scroll and see ;)

**Comment by vimtor** [comment]
Does Raycast markdown support LaTeX?

**Comment by j3lte** [comment]
Nope, but the library that is used (`wikijs`) also doesn't properly render it. The way to probably properly render it is to take the HTML instead, remove the Latex (because the HTML has fallback images with proper rendering), reveal the images and transform the HTML to Markdown. That's the way I would do it (did a little bit of research), but haven't had time to properly build it. Also, in HTML I get a ton more unrelated stuff that shouldn't be in the markdown, so it means we'd have to filter more.
